guides/common/modules/proc_increasing-log-levels.adoc

```diff
@@ -0,0 +1,251 @@
+[id="Increasing_Log_Levels_{context}"]
+= Increasing Log Levels
+
+By default, {Project} comes with `:INFO` level logging enabled.
+You can increase or decrease the log levels on your {Project}.
+
+.Enabling debug level logging on all components
+[options="nowrap" subs="+quotes"]
+----
+# hammer admin logging --all --level-debug
+# {foreman-maintain} service restart
+----
+
+.Enabling debug level logging for a specific component
+[options="nowrap" subs="+quotes"]
+----
+# hammer admin logging --components "_Component_" --level-debug
+----
+
+.Reverting debug level logging to `INFO`
+[options="nowrap" subs="+quotes"]
+----
+# hammer admin logging --all --level-production
+# {foreman-maintain} service restart
+----
+
+.Listing all components and changed configuration files
+----
+# hammer admin logging --list
+-----------|-------------------------------------|-------------------------------------
+COMPONENT  | AUTO-DETECTED BY EXISTENCE OF       | DESTINATIONS
+-----------|-------------------------------------|-------------------------------------
+dhcpd      | /etc/dhcp/dhcpd.conf                | syslog /var/log/dhcpd-debug.log
+postgresql | /var/lib/pgsql/data/postgresql.conf | syslog /var/lib/pgsql/data/pg_log/
+proxy      | /etc/foreman-proxy/settings.yml     | /var/log/foreman-proxy/proxy.log
+qpidd      | /etc/qpid/qpidd.conf                | syslog
+rails      | /etc/foreman/settings.yaml          | /var/log/foreman/production.log
+tomcat     | /etc/candlepin/candlepin.conf       | /var/log/candlepin/ /var/log/tomcat/
+virt-who   | /etc/sysconfig/virt-who             | syslog
+-----------|-------------------------------------|-------------------------------------
+----
+
+== Hammer
+
+You can find the log for Hammer in `~/.hammer/log/hammer.log`.
+
+.Changing the log level
+. Edit `/etc/hammer/cli_config.yml` and set the `:log_level:`:
++
+----
+:log_level: 'debug'
+----
+
+== {SmartProxy}
```

Comment:
```suggestion
== Increasing the logging level on {SmartProxy}
```

How about that?

Reply:
Yes, much better. Can I leave it as "level2 heading" and drop the repetitive `.Changing the log level`?

Reply:
Works for me :)

Reply:
Applied.